tapdb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -172,6 +172,10 @@ CREATE TABLE IF NOT EXISTS assets (
     
     genesis_id INTEGER NOT NULL REFERENCES genesis_assets(gen_asset_id),
 
+    asset_computed_id BLOB UNIQUE,
```

Comment:
Same here, I don't think we need any of these fields. We'll already know the genesis information before we even are sent an asset. That information will always be there, as it must be known in order to properly verify an asset. 

In other words, we shouldn't really need to change this on disk structure. What we're changing is the wire level serialization: where these fields are encoded vs not. Ideally all the higher level logic stays the exact same (or needs minimal changes at least).

Reply:
I think we still need changes for the group key side though, to now store the tapscript root?

Reply:
Specifically a new field in the `asset_groups` table to store tapscript roots.

Reply:
> Specifically a new field in the asset_groups table to store tapscript roots.

Yep agreed this is necessary. I think the other question this leads into is: shoudl we break out the group key derivation info into another sub-table? Today we have the ability to crate and spend form script keys that have script paths, this logic is done primary outside the daemon though. Given that minting right now is something fully encapsulated in the daemon, do we also need to store the leaves as well for the group key tapscript tree, or would we rely again on the caller to handle that?

Reply:
Discussed offline, resolved to just store the root for now.